third-culture-software/bhima
# Prevent null values of unit_cost by Stock Entry form

Prevents deleting unit cost field and make sure that it is a number > 0.    
Fixes #4718.

Suggested testing procedure:
  * Use the sequence outlined in Issue #4718 
  * Try deleting the Unit Cost field.  The modal dialog should report an error and prevent submission.
  * Verify that angular prevents entry of non-numbers
  * Check entering negative value

Translation:
  * Please review French translation of STOCK.ERRORS.MISSING_LOT_UNIT_COST
